content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -16,8 +16,8 @@ description: >-
 
 <!-- overview -->
 
-If you want to control traffic flow at the IP address or port level (OSI layer 3 or 4), then you
-might consider using Kubernetes NetworkPolicies for particular applications in your cluster.
+If you want to control traffic flow at the IP address or port level for TCP, UDP, and SCTP protocols,
+then you might consider using Kubernetes NetworkPolicies for particular applications in your cluster.
```

Comment:
I tried to do that under `## Network traffic filtering`, do you think it would be nice to also add a sentence about undefined behaviour for other protocols here in the beginning of the doc?